Cannot run unit tests

<!-- 
Thanks for contributing to Simplenote! Pick a clear title ("Note editor: emojis not displaying correctly") and proceed.

Please review the FAQs before submitting an issue: https://github.com/Automattic/simplenote-electron/labels/FAQ
-->

#### Steps to reproduce
1. Run a unit test with command `npm run test` or `jest`
2. Found an error `Cannot find module '../' from 'test.jsx'`

#### What I expected
It should start running unit test

#### What happened instead
The test fail and don't run

#### OS version
macOS 10.12.4
node v6.9.2
npm  v3.10.9
jest v20.0.0

#### Screenshot / Video
![screen shot 2560-05-11 at 9 39 55 am](https://cloud.githubusercontent.com/assets/1977772/25929997/cfec6da2-362d-11e7-936d-cf52530a34aa.png)

<!--
PLEASE NOTE
- These comments won't show up when you submit the issue.
- Everything is optional, but try to add as many details as possible.
- If requesting a new feature, explain why you'd like to see it added.
-->

--- a/lib/components/tag-chip/test.jsx
+++ b/lib/components/tag-chip/test.jsx
@@ -2,7 +2,7 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import { mount, shallow } from 'enzyme';
 
-import TagChip from '../';
+import TagChip from './';
 
 describe( 'TagChip', () => {
 	it( 'should select tag when clicked', () => {

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "autoprefixer": "6.5.4",
     "babel-core": "6.21.0",
     "babel-eslint": "7.1.1",
+    "babel-jest": "20.0.3",
     "babel-loader": "6.2.10",
     "babel-preset-es2015": "6.18.0",
     "babel-preset-react": "6.16.0",

